fem/integ/bilininteg_mass_kernels.cpp

```diff
@@ -25,16 +25,13 @@ static void PAMassAssembleDiagonal1D(const int NE,
                                      const int D1D,
                                      const int Q1D)
 {
-   MFEM_VERIFY(D1D <= MAX_D1D, "");
-   MFEM_VERIFY(Q1D <= MAX_Q1D, "");
```

Comment:
Shouldn't we have a check here? This code was just removed, not replaced.

Reply:
Maybe this is fine -- there are no arrays here and sizes are dynamic.

Reply:
That's right, the checks were unnecessary.